webapp/src/components/tables/existingConfigTable.tsx

```diff
@@ -0,0 +1,187 @@
+import React, {useState} from 'react';
+
+import {FormGroup, Table, ButtonGroup, Button, OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+import ActionModal from '../modals/actionModal';
+import DeleteModal from '../modals/deleteModal';
+import ConfigModal from '../modals/configModal';
+
+import {DeleteSvg, EditSvg, ViewSvg} from '../svgIcons/svg';
+
+import './styles.scss';
+
+type Props = {
+    onChange: (config: Configs[]) => void;
+    value: Configs[];
```

Comment:
```suggestion
    configValues: Configs[];
```

Reply:
cannot changes the values as these props are received from backend and needed to be matched